docs/trust-framework-roles/1.0.md

```diff
@@ -0,0 +1,106 @@
+
+Each Member of the Trust Framework must assign two named individuals in the Directory, each responsible for distinct but complementary legal and technical functions required for participation in the Trust Framework and any associated Schemes.
+
+These roles are:
+
+* **Trust Framework Licence Officer (TFLO)** – responsible for legal authority over licensing and consent
+* **Trust Framework Data Officer (TFDO)** – responsible for technical assurance and compliance
+
+Assignment of these roles is mandatory for all Members. If a named individual leaves the organisation or can no longer fulfil the role, the Member must update the Directory without delay to maintain compliance.
+
+The responsibilities and required capabilities of these roles are defined below.
+
+## Dependencies
+
+A Scheme using this specification must also adopt the following specifications:
+
+* [Generic Sensitivity Classes](../generic-sensitivity-classes/1.0.md) (1.0 or later)
+
+## Trust Framework Licence Officer (TFLO)
+
+The Trust Framework Licence Officer has legal authority to make decisions on data sharing, data use, and consent on behalf of the Member organisation. The TFLO is accountable for ensuring that data published or consumed under the Trust Framework complies with relevant licensing terms and consent conditions.
+
+A TFLO:
+
+* Has authority to sign or approve data licences (e.g. Open Data, Shared Data) on behalf of the Member.
+* Provides legal consent for the use of personal or consumer data where required.
+* Is accountable for compliance with all data licensing and consent terms, as both data provider and data consumer.
+* Ensures licences are reviewed, maintained, or revoked as necessary.
+
+### Responsibilities by Sensitivity Class
+
+#### Sharing data
+
+When sharing Open data (IB1-O) the TFLO:
+
+* _Authorises the allocation of an appropriate Open Data licence to an Open dataset published within the context of the Trust Framework_
+* _Signs_
+
+When sharing Shared data (IB1-SA or IB1-SB) the TFLO:
+
+* _Determines which preemptive Shared data licence(s) is/are allocated to Shared datasets shared within the context of the Trust Framework and any associated Schemes_
+* _Signs the preemptive Shared data licence(s) to confirm its/their allocation to a particular dataset for use within the context of the Trust Framework_
+* _If required, reconfirms or adjusts licences at appropriate intervals (e.g. in the case of time-bound licence allocation)_
+
+When sharing Personal or Consumer data (IB1-SP), the TFLO:
+
+* _Reviews the terms and conditions of providing permission/consent to share data_
+* _Determines whether the terms and conditions are acceptable to the member organisation_
+* _Provides permission/consent to share the data_
+* _Manages the organisation’s consent/permission (e.g. reconfirming permission at time-bound intervals, revoking consent if required) for sharing the dataset_
+
+#### Using Data
+
+When using Open data, the TFLO:
```

Comment:
When using Open data *within the context of a Trust Framework and its associated Schemes*, the TFLO:

Same goes for the IB1-SA/SB and IB1-SP info. Just for consistency with the data publishing descriptions above